services/webinar/api.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { Headers } from 'node-fetch';
+
+import { fetchWithBody, apiBase } from './utils';
+
+export function registerParticipant({
+  firstName,
+  lastName,
+  email,
+  organizerKey,
+  webinarKey,
+  accessToken,
+}: {
+  firstName: string,
+  lastName: string,
+  email: string,
+  organizerKey: string,
+  webinarKey: string,
+  accessToken: string,
+}) {
+  const url = `${apiBase}/G2W/rest/organizers/${organizerKey}/webinars/${webinarKey}/registrants`;
+  const query = '?resendConfirmation=true';
+
+  const headers = new Headers();
+  headers.append('Accept', 'application/vnd.citrix.g2wapi-v1.1+json');
+  headers.append('Authorization', accessToken);
+  headers.append('Content-Type', 'application/json');
+
+  return fetchWithBody(url + query, {
+    method: 'POST',
+    headers,
+    body: JSON.stringify({
+      firstName,
+      lastName,
+      email,
+    }),
+  }).then(({ response, responseBody }) => {
+    if (response.status === 201 || response.status === 409) {
```

Comment:
We could add some constant definitions for these status codes (eg `created` for 201 and `conflict` for 409) so that it's easier to parse at first glance. I admit I had to look up what those status codes meant 🙄  I assume that these are the only status codes that we care about?